src/widgets/SpinBox/VNSpinBox.ts

```diff
@@ -12,13 +12,20 @@ import { PropSetters, Prop } from '../../renderer/patchProp';
  * ```html
  * <template>
  *  <vn-view>
- *    <vn-text>How many slices of cake would you like?</vn-text>
- *    <vn-spinbox />
+ *    <vn-text>How many slices of cake ($2) would you like?</vn-text>
+ *    <vn-spinbox v-model="spinBoxValue"/>
+ *    <vn-text>Total cost: ${{spinBoxValue * 2}}</vn-text>
  *  </vn-view>
  * </template>
  *
  * <script>
- *  export default { }
+ *   export default {
+ *     data() {
```

Comment:
Could you please use the `setup` API instead of the `data`?
You can check out a sample here: https://github.com/nodegui/vue-nodegui/blob/master/src/widgets/LineEdit/VNLineEdit.ts

Reply:
Sorry about that, changed it now 😅